changes/1846.bugfix.rst

```diff
@@ -0,0 +1 @@
+Created validation for document types with error handling.
```

Comment:
The release note should be written as if it were a line in a release note, describing the new feature, rather than a description of the commit:
```suggestion
Document type declarations are now fully validated.
```